src/authentication/controllers/fetchSlackUserStatus.ts

```
@@ -0,0 +1,26 @@
+    res: Response<SlackUserStatusResponse>,
+    next: NextFunction
+) => {
+    const email = req.query.userEmail as string;
+
+    if (!email) {
+        
+        return next(new CustomError('User email must be provided', 400))
+    }
+    try {
+        const user = await fetchSlackUser({ email });
+        const isActive = user? true: false;
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)